### PR TITLE
Project documentation

### PR DESCRIPTION
**Pull request by 12mihai05**
### Pull Request Overview

This pull request adds the hardware description and kicad schematics.

### Author

Signed-off-by: Pasaroiu Mihai <pasaroiumihai@yahoo.com>

### CODE REVIEW

File: docs/project/2024/mihai.pasaroiu/project.kicad_sch
Comment (alexandruradovici) [inline docs/project/2024/mihai.pasaroiu/project.kicad_sch:1]: Please move this file to your project repository.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/mihai.pasaroiu/project.kicad_sch
Comment (12mihai05) [reply docs/project/2024/mihai.pasaroiu/project.kicad_sch:1]: i moved it.

Comment (alexandruradovici) [reply docs/project/2024/mihai.pasaroiu/project.kicad_sch:1]: The file is still in this branch.

**Review by alexandruradovici — DISMISSED** [review]

File: docs/project/2024/mihai.pasaroiu/project.kicad_sch
Comment (12mihai05) [reply docs/project/2024/mihai.pasaroiu/project.kicad_sch:1]: i removed it, also i have a question do we need to put images with the build of the hardware, not only the schematic, in the documentation?

**Comment by 12mihai05** [comment]
do i need to put images in the documentation with the build hardware, not only the schematics in kicad?

**Comment by alexandruradovici** [comment]
Yes, please put some images.

**Review by alexandruradovici — DISMISSED** [review]

**Comment by alexandruradovici** [comment]
Please fix the conflicts.

**Review by alexandruradovici — APPROVED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by TeodoraMiu — APPROVED** [review]

**Pull request merged**